.github/workflows/track-pr.yml

```diff
@@ -0,0 +1,22 @@
+name: track-pr
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  trackpr:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PR: ${{ toJSON(github.event.pull_request) }}
+      PR_ID: ${{ github.event.pull_request.node_id }}
+      GH_TOKEN: ${{ secrets.GH_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.19'
+
+      # Add PR to project board.
+      - name: "Track PR"
+        if: ${{ success() }}
```

Comment:
what is this "if success"?

Reply:
yeah, don't need it as we don't have any previous step. Btw, `success()` returns true when none of the previous steps have failed or been canceled. https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions